robottelo/cli/factory.py

```diff
@@ -648,7 +648,7 @@ def make_registry(options=None):
         u'description': None,
         u'name': gen_string('alphanumeric'),
         u'password': None,
-        u'url': DOCKER_0_EXTERNAL_REGISTRY,
+        u'url': settings.docker.external_registry_1,
```

Comment:
Robottelo properties is able to read list properties. I will open an issue to improve this to use the list notation instead of naming 1, 2, 3, ...